client/models.py

```diff
@@ -76,7 +78,9 @@ class VolunteerSchedule(Timestampable):
 
 class Volunteer(Timestampable):
```

Comment:
The following fields should be optional (either with blank=True, null=True or with a default value):

* areas
* languages
* available_starday
* keep_mandatory_worker_children
* guiding
* hearing_way

The following fields should have null=True (they currently have blank=True):

* notes
* organization
* neighborhood